### PR TITLE
Remove all in-game mentions of 1.5x unhappiness for puppeted cities

### PR DESCRIPTION
**Pull request by Ouaz**
This has been removed in 87aa5bc (regular happiness now).

### CODE REVIEW

File: android/assets/jsons/translations/template.properties [hunk 1]
```diff
@@ -1232,7 +1232,7 @@ Annex =
 Annexed cities become part of your regular empire. = 
 Their citizens generate 2x the unhappiness, unless you build a courthouse. = 
 Puppet = 
-Puppeted cities do not increase your tech or policy cost, but their citizens generate 1.5x the regular unhappiness. = 
+Puppeted cities do not increase your tech or policy cost, but their citizens generate the regular unhappiness. =
```
Comment (yairm210) [inline android/assets/jsons/translations/template.properties:1235]: No need for " but their citizens generate the regular unhappiness", we can just remove that

Comment (Ouaz) [reply android/assets/jsons/translations/template.properties]: Done. Sorry for the multiple commits, but I'm lazy to clone Unciv just for that.^^
Should I remove this part from the tutorial too? I think it useful for newcomers to know that puppet a city is still heavy on happiness, if not planned before.

File: android/assets/jsons/Tutorials.json [hunk 2]
```diff
@@ -108,7 +108,7 @@
...
+      "\nLiberating the city will return it to its original owner, giving you a massive diplomatic boost with them!\n\nAnnexing the city will give you full control over it, but also increase the citizens' unhappiness to 2x!\nThis can be mitigated by building a courthouse in the city, returning the citizen's unhappiness to normal.\n\nPuppeting the city will mean that you have no control on the city's production.\nThe city will not increase your tech or policy cost, but its citizens will generate the regular unhappiness.\nA puppeted city can be annexed at any time, but annexed cities cannot be returned to a puppeted state!\n\nRazing the city will lower its population by 1 each turn until the city is destroyed!\nYou cannot raze a city that is either the starting capital of a civilization or the holy city of a religion."
```
Comment (OptimizedForDensity) [inline android/assets/jsons/Tutorials.json:111]: Same as Yair's above comment: "\nThe city will not increase your tech or policy cost, but its citizens will generate the regular unhappiness."

Comment (Ouaz) [reply android/assets/jsons/Tutorials.json]: Ok, I removed entirely mention of unhappiness for puppeted cities, as asked.

**Pull request merged**